Lib/ufo2ft/outlineCompiler.py

```diff
@@ -1050,6 +1057,123 @@ def setupTable_meta(self):
                     f"public.openTypeMeta '{key}' value should be bytes or a string."
                 )
 
+    def _bboxWidth(self, glyph):
+        bbox = self.glyphBoundingBoxes[glyph]
+        if bbox is None:
+            return 0
+        return bbox.xMax - bbox.xMin
+
+    def _bboxHeight(self, glyph):
+        bbox = self.glyphBoundingBoxes[glyph]
+        if bbox is None:
+            return 0
+        return bbox.yMax - bbox.yMin
+
+    def setupTable_MATH(self):
```

Comment:
we should document the format of these lib keys somewhere

Reply:
also maybe we could propose these libs to the UFO spec and make them "public.*"

Reply:
I can try to propose them to UFO spec, but there are several fonts that use these private keys already (one of them is Noto Sans Math which we want to build cleanly with fontmake, instead of the current brittle build that use TTX dump of the MATH table).

If the UFO changes happen and go through, we can deprecate these or even drop completely.